code/renderers/vue3/template/stories/preview.js

```diff
@@ -1,25 +1,12 @@
-import type { App, Plugin } from 'vue';
-import type { StoryContext } from '@storybook/csf';
 import { global as globalThis } from '@storybook/global';
 // eslint-disable-next-line import/no-extraneous-dependencies
-import { setup, type VueRenderer } from '@storybook/vue3';
+import { setup } from '@storybook/vue3';
 
-declare module 'vue' {
-  interface ComponentCustomProperties {
-    $greetingMessage: (key: string) => string;
```

Comment:
It seems like this change broke the `check` command, as `$greetingMessage` is not augmented anymore when the story is included in `vue3-vite-default-ts/`:

```
template/stories_vue3-vite-default-ts/GlobalSetup.vue(3,23): error TS2339: Property '$greetingMessage' does not exist on type '{ $: ComponentInternalInstance; $data: {}; $props: Partial<{}> & Omit<Readonly<ExtractPropTypes<{}>> & VNodeProps & AllowedComponentProps & ComponentCustomProps, never>; ... 10 more ...; $watch<T extends string | ((...args: any) => any)>(source: T, cb: T extends (...args: any) => infer R ? (args_0: R, args_1: R) => ...'.
template/stories_vue3-vite-default-ts/GlobalSetup.vue(4,26): error TS2339: Property '$greetingMessage' does not exist on type '{ $: ComponentInternalInstance; $data: {}; $props: Partial<{}> & Omit<Readonly<ExtractPropTypes<{}>> & VNodeProps & AllowedComponentProps & ComponentCustomProps, never>; ... 10 more ...; $watch<T extends string | ((...args: any) => any)>(source: T, cb: T extends (...args: any) => infer R ? (args_0: R, args_1: R) => ...'.
template/stories_vue3-vite-default-ts/GlobalSetup.vue(5,28): error TS2339: Property '$greetingMessage' does not exist on type '{ $: ComponentInternalInstance; $data: {}; $props: Partial<{}> & Omit<Readonly<ExtractPropTypes<{}>> & VNodeProps & AllowedComponentProps & ComponentCustomProps, never>; ... 10 more ...; $watch<T extends string | ((...args: any) => any)>(source: T, cb: T extends (...args: any) => infer R ? (args_0: R, args_1: R) => ...'.
```